packages/cli/package.json

```diff
@@ -16,7 +16,7 @@
     "test": "mocha -r ts-node/register \"test/{,!(fixtures)/**}/*.test.ts\"",
     "mocha": "mocha -r ts-node/register",
     "bigtest": "ts-node src/index.ts",
-    "prepack": "tsc --outdir dist --module commonjs && chmod +x dist/bin/bigtest.js"
+    "prepack": "tsc --outdir dist --module commonjs && node chmod.js 755 dist/bin/bigtest.js"
```

Comment:
Why do we even need to `chmod` here? Shouldn't yarn/npm take care of this for us?

Reply:
You will need `chmod` for shebangs etc.

you can  use the `--chmod` switch of `git update-index` though to commit

e.g.

```bash
git update-index --chmod=755 --add dist/bin/bigtest.j
```

Reply:
Never new that was a even a thing!